scripts/respeaker_node.py

```diff
@@ -7,6 +7,7 @@
 import usb.core
 import usb.util
 import pyaudio
+import wave
```

Comment:
this should be removed again as well, right? that was for the wav file you used for debugging?